pkg/apis/api/v1alpha1/astarte_types.go

```diff
@@ -138,14 +138,21 @@ func (a *AstarteComponent) DockerImageName() string {
 	return "astarte_" + a.String()
 }
 
-// ServiceName returns the Kubernetes Service Name associated to this Astarte component,
-// if any, otherwise returns an empty string.
-// This will return a meaningful value only for API components or the Dashboard.
+// ServiceName returns the Kubernetes Service Name associated to this Astarte component.
 func (a *AstarteComponent) ServiceName() string {
+	return a.DashedString()
+}
+
+// ServiceRelativePath returns the Kubernetes Service Name associated to this Astarte component,
```

Comment:
"returns the relative path where the service will be served by the Astarte Voyager Ingress" or something like that